Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -248,5 +248,25 @@ public static void Unhover ([NotNull] this ElementScope scope)
 
       Cursor.Position = new Point (0, 0);
     }
+
+    /// <summary>
+    /// Gets an attribute from the scope or throws a <see cref="MissingHtmlException"/> if it could not be found. Wraps <see cref="ElementScope.this"/>.
+    /// </summary>
+    /// <remarks>
+    /// Accessing HTML attributes through <see cref="ElementScope.this"/> returns null if the attribute does not exist.
+    /// This might not be the desired behavior in some cases, thus offering this extension method.

```

Comment:
Sentence doesn't make sense to me.

Reply:
Could you elaborate a little? I meant to say that returning `null` might not always be the desired behavior and offering this method provides a different behavior.  What did you get from the text instead?

Reply:
also: <see langword=null>

As to the question: I don't know. yesterday I wasn't able to understand what it was trying to tell me. Maybe...replace second sentence with 

By using cref=GetAttribute instead, a cref=MissingHTMLexception is thrown if the HTML attribute does not exist on the scope.

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/07ec2f7cdccd2a2b833b0c64cd692af7fe4d79f0.